packages/manager/modules/pci/src/projects/project/instances/instances.service.js

```diff
@@ -129,6 +135,12 @@ export default class PciProjectInstanceService {
       }));
   }
 
+  static transformCapabilities(capabilities) {
+    return reduce(capabilities, (flavorCapabilities, currentCapability) => set(
```

Comment:
I think we should avoid using `set` on an object when it's possible and instead return an updated copy of the original object. 
In that particular case it shouldn't have much side effects but in others it can have

Reply:
The `set` call is removed and replaced by a new object return